plugins/codemodder-plugin-maven/src/test/java/io/codemodder/plugins/maven/operator/POMOperatorTest.java

```diff
@@ -519,4 +519,68 @@ void testCaseWithEmptyElementFromCustomer() throws Exception {
         resultPomAsString.contains(
             "                <DependencyConvergence></DependencyConvergence>"));
   }
+
+  @Test
+  void insert_should_fail_because_dependency_exists() throws Exception {
+    Dependency dependencyToUpgrade =
+        new Dependency("org.dom4j", "dom4j", "2.0.2", null, null, null);
+
+    ProjectModel context =
+        performInsert(
+            "pom-case-3-insert-fails",
+            ProjectModelFactory.load(POMOperatorTest.class.getResource("pom-case-3.xml"))
+                .withDependency(dependencyToUpgrade)
+                .withSkipIfNewer(true));
+
+    Assert.assertTrue("Original POM File is Dirty", !context.getPomFile().getDirty());
```

Comment:
This logic is weird, can we write it so it's not a double negative?
```suggestion
    Assert.assertFalse("Original POM File is Dirty", context.getPomFile().getDirty());
```

Reply:
it is weird... i got confused at some point while i was having some issues